docs/types/docs.d.ts

```diff
@@ -19,3 +19,5 @@ declare module 'docs/src/modules/components/HighlightedCode' {
 }
 
 declare module '@trendmicro/react-interpolate';
+
+declare module 'react-imask';
```

Comment:
Having this is kinda misleading because the examples don't mention that one has to override `react-imask` types to make it work. Otherwise there is a type error as mentioned in https://github.com/mui/material-ui/issues/30854